src/payeedialog.cpp

```diff
@@ -457,9 +459,13 @@ mmPayeeDialog::mmPayeeDialog(wxWindow* parent, bool payee_choose, const wxString
     ColName_[PAYEE_WEBSITE] = _t("Website");
     ColName_[PAYEE_NOTES] = _t("Notes");
     ColName_[PAYEE_PATTERN] = _t("Match Pattern");
-
+    ColName_[PAYEE_USED] = _t("Used");
```

Comment:
Please add a keyboard shortcut.

Reply:
For what purpose? The other columns have no shortcuts, too.

Reply:
I made a mistake. Sorry.